cluster-autoscaler/processors/datadog/pods/transform_local_data_test.go

```diff
@@ -92,6 +98,26 @@ func TestTransformLocalDataProcess(t *testing.T) {
 			[]*corev1.PersistentVolumeClaim{},
 			[]*corev1.Pod{},
 		},
+
+		{
+			"topolvm provisioner is using proper storage capacity value",
+			[]*corev1.Pod{buildPod("pod1", testEmptyResources, testEmptyResources, "pvc-1", "pvc-2")},
+			[]*corev1.PersistentVolumeClaim{
+				buildPVC("pvc-1", testRemoteClass),
+				buildPVCWithStorage("pvc-2", storageClassNameTopolvm, "100Gi"),
+			},
+			[]*corev1.Pod{buildPod("pod1", testTopolvmResources, testTopolvmResources, "pvc-1")},
+		},
+
+		{
+			"one pvc will override the other",
+			[]*corev1.Pod{buildPod("pod1", testEmptyResources, testEmptyResources, "pvc-1", "pvc-2")},
+			[]*corev1.PersistentVolumeClaim{
+				buildPVCWithStorage("pvc-1", storageClassNameTopolvm, "100Gi"),
+				buildPVC("pvc-2", storageClassNameLocal),
```

Comment:
Should it be possible to have multiple PVC's?

Reply:
we should be able to have 2 lvm pvc on the same node, we can't have "legacy" local-data and lvm at the same time on the nodes though

Reply:
Sorry I've added the comment at the worst place. 
The question was more like if one Pod can have multiple topolvm volume claims simultaneously because this would mean we have to add up the total capacity needed. Otherwise, the latter will override the first.

Reply:
hmm good question, I don't think we want to support that right now but there might be use cases